ucx.spec.in

```diff
@@ -92,10 +92,10 @@ Provides header files and examples for developing with UCX.
 %define _with_arg()   %{expand:%%{?with_%{1}:--with-%{2}}%%{!?with_%{1}:--without-%{2}}}
 %define _enable_arg() %{expand:%%{?with_%{1}:--enable-%{2}}%%{!?with_%{1}:--disable-%{2}}}
 %configure --disable-optimizations \
-           --disable-logging \
-           --disable-debug \
-           --disable-assertions \
-           --disable-params-check \
+           %{!?devel:--disable-logging} \
+           %{!?devel:--disable-debug} \
+           %{!?devel:--disable-assertions} \
+           %{!?devel:--disable-params-check} \
```

Comment:
pls also **enable** following options for debug build:
```
--with-valgrind
--enable-profiling
--enable-frame-pointer
--enable-stats
--enable-fault-injection
--enable-debug-data
--enable-mt
```

also add `BuildRequires: valgrind-devel` in case of debug

Reply:
Done